docs/src/main/tut/auth.md

```diff
@@ -20,12 +20,10 @@ import org.http4s._
 
 case class User(id: Long, name: String)
 
-def addInput[I, T](k: Service[I, T]): Service[I, (T, I)] =
-  Kleisli(input => k.run(input).map(user => (user, input)))
-
-val authUser: Service[Request, User] = Kleisli(_ => Task.delay(???))
+// Needs to be Kleisli for the type inference of &&& to work.
```

Comment:
I wonder whether SI2712-fix handles this, though I don't want to make it a requirement.